### PR TITLE
added support for -t option

### PR DESCRIPTION
**Pull request by andy5995**
* comma separated list of tags now accepted
* any spaces in the comma-separated list will not render correctly

### CODE REVIEW

**Comment by ismnoiet** [comment]
Thank you @andy5995 for your pull request, I will test it tomorrow or the day after and get back to you.

**Comment by andy5995** [comment]
Sounds good.

I haven't tested it against your dev branch. I didn't realize that branch existed until after I'd made my changes.

I tested it by copying my revision to /lib/node_modules/jekyll-posts-generator/bin/jposts after I had installed the version available through npm.

I wasn't sure how to test a working copy. I assume it's a matter of editing the path so it will use all the files in my source tree directory.

Anyway, I did the PR against your 'dev' branch.

File: bin/jposts [hunk 1]
```diff
@@ -8,32 +8,36 @@ var defaultHeader = {layout: 'post',title: '',tags: '',categories: ''}
...
-  // .option('-d, --date', 'Get current Date(format dd/mm/yyyy)')  
-  // .option('-t, --tags', 'Set the current article tags')  
+  // .option('-d, --date', 'Get current Date(format dd/mm/yyyy)')
+  // .option('-t, --tags', 'Set the current article tags')
 
 program
   .command('create-post <postTitle>')
   .description('Create a post')
   .option('-c, --categories <items>', 'Specify the categor(y/ies) for the current article \n                          example: jposts create <postName> -c category1,category2,...')
+  .option('-t, --tags <items>', 'Specify the categor(y/ies) for the current article \n                          example: jposts create <postName> -t tag1,tag2,...')
```
Comment (ismnoiet) [inline bin/jposts]: Please change `Specify the categor(ies)` to `Specify the tag(s)`.

File: bin/jposts [hunk 2]
```diff
@@ -8,32 +8,36 @@ var defaultHeader = {layout: 'post',title: '',tags: '',categories: ''}
...
       } else {
         defaultHeader.title = postTitle
         // console.log('file does not exist')
-         
+
         if (options.categories) {
           defaultHeader.categories = options.categories.split(',')
         }
+        if (options.tags) {
+          defaultHeader.tags = options.tags.split(',')
```
Comment (ismnoiet) [inline bin/jposts]: Please, change `options.tags.split(',')` to `options.tags.replace(/[\s]{1,}/g, '').split(',')` so tags with white spaces will be rendered correctly.

**Review by ismnoiet — CHANGES_REQUESTED** [review]
@andy5995 Please add the requested changes and push the commit so I can merge your PR.

**Comment by andy5995** [comment]
I rebased, corrected some errors, and I think we have as clean a commit log as we're gonna get! :)

**Pull request merged**